ReadMe.md

```diff
@@ -101,7 +101,10 @@ command line parameters on the first run:
 - `install` - build and install all public artifacts into local maven repository
 - `coreLibsTest` - build and run stdlib, reflect and kotlin-test tests
 - `gradlePluginTest` - build and run gradle plugin tests
-- `compilerTest` - build and run all compiler tests
+- `compilerTest` - build and run all compiler tests. These tests are not actively maintained, so only check the tests you are interested in.
+  * A lot of tests check the functionality related to the latest (not necessarily stable) JDK, so make sure the project is built using that.
```

Comment:
Is there really _a lot of tests_? And do they check the _latest_ JDK? AFAIK we have a few tests on JDK 15 and that is it. The latest stable is JDK 18.

Also, the statement that project should be built with the latest JDK to make those tests pass is incorrect AFAIK, you only need to have it installed and be referenced with the environment variable `JDK_15_0`, something which could rather be explained in the "Build environment requirements" section above. (Or this requirement might have even become obsolete already when we started using JDK toolchains via Gradle -- I don't know.)